paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -417,4 +426,50 @@ internal sealed interface PaymentSheetScreen {
             ManageOneSavedPaymentMethodUI(interactor = interactor)
         }
     }
+
+    class CvcRecollection(
+        private val args: CvcRecollectionContract.Args,
+        private val interactor: CvcRecollectionInteractor
+    ) : PaymentSheetScreen, Closeable {
+        override val showsBuyButton = false
+        override val showsContinueButton = false
+
+        override fun topBarState(): StateFlow<PaymentSheetTopBarState?> {
+            return stateFlowOf(
+                PaymentSheetTopBarStateFactory.create(
+                    hasBackStack = true,
+                    isLiveMode = false,
+                    editable = PaymentSheetTopBarState.Editable.Never,
+                )
+            )
+        }
+
+        override fun title(isCompleteFlow: Boolean, isWalletEnabled: Boolean): StateFlow<ResolvableString?> =
+            stateFlowOf(null)
+
+        override fun showsWalletsHeader(isCompleteFlow: Boolean) = stateFlowOf(false)
+
+        @Composable
+        override fun Content(modifier: Modifier) {
+            val viewModel = viewModel<CvcRecollectionViewModel>(
+                factory = CvcRecollectionViewModel.Factory(args)
+            )
+            val state = viewModel.viewState.collectAsState()
```

Comment:
rather than creating the view model here, we would usually just have a composable that we could pass the interactor into. We often can create the interactor using a view model, but that would happen outside of the actual screen code. E.g. you can see how we do this in the ManageOneSavedPaymentMethod screen. 

This would probably require refactoring the `CvcRecollectionScreen` to take the interactor you defined as its sole parameter